packages/ui-kit/src/components/ChipButton/ChipButton.tsx

```diff
@@ -0,0 +1,28 @@
+import { VFC, HTMLAttributes } from "react";
+import clsx from "clsx";
+
+import styles from "./ChipButton.module.css";
+
+export interface ChipButtonProps extends HTMLAttributes<HTMLSpanElement> {
+  label: string;
+  active?: boolean;
+}
+
+export const ChipButton: VFC<ChipButtonProps> = ({
```

Comment:
In storybook basic chip button doesn't have border (on designs it does)

Reply:
Good point, fixed.